static/app/components/events/interfaces/breadcrumbs/index.tsx

```diff
@@ -316,19 +330,16 @@ function BreadcrumbsContainer({data, event, organization, route, router}: Props)
           <h3>{t('Breadcrumbs')}</h3>
         </GuideAnchor>
       }
-      actions={
-        <StyledSearchBarAction
-          placeholder={t('Search breadcrumbs')}
-          onChange={handleSearch}
-          query={searchTerm}
-          filterOptions={filterOptions}
-          filterSelections={state.filterSelections}
-          onFilterChange={handleFilter}
-        />
-      }
+      actions={!replay && searchBar}
       wrapTitle={false}
       isCentered
     >
+      {replay && (
+        <Fragment>
+          {replay}
+          {searchBar}
+        </Fragment>
+      )}
```

Comment:
Don't use `&&` unless the left side is a boolean.

As it is, `replay = 0` is perfectly allowed by typescript, and will render a 0 onto the page. Everyone's probably seen that before and tried to sort out the where the extra is coming from. I just use ternary all the time and never have an issue. Also update the `&&` above on line 333 to start creating the habit.

Might also want to change